src/agent/solana.rs

```diff
@@ -144,6 +144,8 @@ mod key_store {
         pub program_key_path:     PathBuf,
         /// Path to the public key of the root mapping account, relative to the root
         pub mapping_key_path:     PathBuf,
+        /// Path to the public key of the accumulator program, relative to the root.
+        pub accumulator_key_path: Option<PathBuf>,
```

Comment:
(it's confusing that these pubkeys are in separate files as opposed to simply being strings in the config, but this is consistent with the other keys so it's fine)